api/server.js

```diff
@@ -0,0 +1,18 @@
+const cors = require('cors');
+const express = require('express');
+const app = express()
+const port = 8000
```

Comment:
server by default should be using port 3000 it is better practice. If you want it to work post deployment you should use below: 

const port = process.env.PORT || 3000;

if you look at your app.js the comments are there.  this file should provide the implementation for server functionality and give it to the app.